app/components/choice-picker/template.hbs

```diff
@@ -0,0 +1,9 @@
+<ul class="choice-list">
```

Comment:
we might be able to eve get rid of the list?

Reply:
wanna try?